modules/version.js

```diff
@@ -0,0 +1,20 @@
+var exec = require('child_process').exec;
+
+var hash;
+
+module.exports.init = function(b) {
+	exec('git rev-parse HEAD', function(error, stdout, stderr) {
+		if(error != null) {
+			hash = null;
+			console.err("git's broken yo");
+		}
+		hash = stdout;
+		console.log(hash);
+	});
+}
+
+module.exports.run = function(r, parts, reply) {
+	reply(hash);
+}
+
+module.exports.commands = ['version', 'head', 'hash'];
```

Comment:
Not a fan of having this many commands for it. Pick one, preferably with a long and inconvenient name :)